blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/MultiBranchPipelineImpl.java

```diff
@@ -264,8 +264,8 @@ private int countJobs(boolean pullRequests) {
         return counter;
     }
     private boolean isPullRequest(Job job) {
-        SCMHead head = SCMHead.HeadByItem.findHead(job);
-        return head != null && head.getAction(ChangeRequestAction.class) != null;
+        // TODO probably want to be using SCMHeadCategory instances to categorize them instead of hard-coding for PRs
```

Comment:
@i386 No. The recommended display order is by localized display name and the default category is always present and always has the `getName()` of `default`